ui/raidboss/popup-text.ts

```diff
@@ -707,8 +707,12 @@ export class PopupText {
     const filterEnabled = (trigger: LooseTrigger) => !('disabled' in trigger && trigger.disabled);
     const allTriggers = orderedTriggers.asList().filter(filterEnabled);
 
-    this.triggers = allTriggers.filter(isRegexTrigger);
-    this.netTriggers = allTriggers.filter(isNetRegexTrigger);
+    this.triggers = allTriggers.filter(
+        (t) => isRegexTrigger(t) && !isRaidbossLooseTimelineTrigger(t),
```

Comment:
I think of them as three mutually exclusive types, where RegexTrigger and TimelineTrigger happen to unfortunately share the key property `regex` that's used to identify them.

As for if the `isRegexTrigger/isNetRegexTrigger` type guards should check for this directly, I think it could go either way, but bearing in mind what you said previously about type guard length, I opted to not include this check in the type guards.

Reply:
Ok, if you also think about them as three mutually exclusive types, then that seems fine for a typeguard.  That's still a one-liner.

Reply:
Made this adjustment. I think that's what you were suggesting?